jib-cli/src/main/java/com/google/cloud/tools/jib/cli/CommonCliOptions.java

```diff
@@ -244,6 +244,15 @@
   @SuppressWarnings("NullAway.Init") // initialized by picocli
   private boolean serialize;
 
+  @CommandLine.Option(
+      names = "--image-metadata-out",
+      paramLabel = "<path-to-json>",
+      description =
+          "A path to the json file that should contain image metadata (for example, digest, id and tags) after build is"
+              + "complete.")
```

Comment:
Nit: should this be
```suggestion
          "path to the json file that should contain image metadata (for example, digest, id and tags) after build is"
              + "complete")
```
?